core/matching/side.go

```diff
@@ -220,10 +220,6 @@ func (s *OrderBookSide) amendOrder(orderAmend *types.Order) (int64, error) {
 // if removeOrders is set to True then the relevant orders also get removed.
 func (s *OrderBookSide) ExtractOrders(price *num.Uint, volume uint64, removeOrders bool) []*types.Order {
 	extractedOrders := []*types.Order{}
-	if volume == 0 {
```

Comment:
I'm assuming a bad merge has removed this. Same for the unit test below.

Reply:
super weird.